samples/sensor/dps310/sample.yaml

```diff
@@ -0,0 +1,12 @@
+sample:
+  name: DP310 Sensor sample
+tests:
+  sample.sensor.dps310:
+    harness: console
+    tags: sensors
+    depends_on: i2c dps310
```

Comment:
Unless you add `dps310` to the `supported` list in a board, this sample will never build in CI.

Reply:
Where do I have to add it? The sample should run on any board that has the Arduino I2C interface defined.

Reply:
I guess it's arbitrary in this case. Do we have a rule for this @MaureenHelm, @galak, @pabigot ?

Reply:
Maybe `depends_on: i2c arduino_i2c`?

Reply:
Changed to `depends_on: i2c arduino_i2c`